help/faq/texprobs.md

```diff
@@ -0,0 +1,88 @@
+# Why doesn't my processed TeX submission look the way I expected it?
+
+There are a number of common reasons why the processed version of TeX submissions to arXiv may not appear the same as it does when processed locally. In most cases there are fixes that can be applied and the submission replaced to correct the problems.
+
+*   [Lines in the table of contents don't wrap](#toc_links)
+*   [Fuzzy fonts in PDF](#fuzzy_pdf)
+*   [PostScript figures lose quality in the arXiv-generated PDF](#distiller_params)
+*   [Extra white space around figures covers up some of the text](#powerpoint)
+*   [Why don't the labels for my figures appear when I use Feynmf?](feynmf#nolabel)
+*   [Why are the margins different? Why is my text truncated?](dvips)
+*   [Why does my paper give the wrong date?](today)
+*   [Why do my citations appear in long form \[1,2,3,4\] instead of short form \[1-4\]?](citelinks)
+*   [Why do my user defined symbols (e.g., \\i, \\l, \\L, \\o, \\ae, etc.) display incorrectly in the processed document?](pd1enc)
+*   [Why are there problems with my postscript file?](/help/faq/psbad)
+*   [Why are some pages in the generated PDF file rotated and how do I avoid this?](/help/faq/pdfrotate)
```

Comment:
In local build the /help/... links don't render correctly, but I believe that they do when posted to the server.

Reply:
changed to relative links